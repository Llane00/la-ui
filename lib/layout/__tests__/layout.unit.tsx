import renderer from 'react-test-renderer'
import React from 'react'
import Layout from '../layout';
import Header from '../header';
import Sider from '../sider';
import Content from '../content';
import Footer from '../footer';

describe('layout', () => {
    it('layout组件快照测试', ()=>{
      const json = renderer.create(<Layout>
      <Header>header</Header>
      <Layout>
        <Sider>sider</Sider>
        <Content>content</Content>
      </Layout>
      <Footer>footer</Footer>
    </Layout>).toJSON()
      expect(json).toMatchSnapshot()
    })
})
