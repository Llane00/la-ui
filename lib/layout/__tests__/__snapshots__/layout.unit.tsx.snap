// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout layout组件快照测试 1`] = `
<div
  className="la-ui-layout"
>
  <div
    className="la-ui-layout-header"
  >
    header
  </div>
  <div
    className="la-ui-layout la-ui-layout-hasSider"
  >
    <div
      className="la-ui-layout-sider"
    >
      sider
    </div>
    <div
      className="la-ui-layout-content"
    >
      content
    </div>
  </div>
  <div
    className="la-ui-layout-footer"
  >
    footer
  </div>
</div>
`;
